#ifndef IMPLAYER_H
#define IMPLAYER_H

#include <QMediaPlayer>
#include <QMediaPlaylist>
#include <QVideoWidget>
#include <QFileDialog>
#include <QFile>
#include <QFileInfo>
#include <QDir>
#include <QListWidgetItem>
#include <QMap>
#include <QEvent>
#include <QMouseEvent>

#include "RedLoong/RedLoong.h"

#endif // IMPLAYER_H
